solr/core/src/test/org/apache/solr/search/TestReload.java

```diff
@@ -36,13 +36,13 @@ public void testGetRealtimeReload() throws Exception {
 
     assertU(commit("softCommit","true"));   // should cause a RTG searcher to be opened
 
-    assertJQ(req("qt","/get","id","1")
+    assertJQ(req("qt","/get","id","1", "fl", "id,_version_")
         ,"=={'doc':{'id':'1','_version_':" + version + "}}"
     );
 
     h.reload();
 
-    assertJQ(req("qt","/get","id","1")
+    assertJQ(req("qt","/get","id","1", "fl", "id,_version_")
```

Comment:
_root_ is also added to the doc so I added fl param to filter it,
since, IMO, testing _root_ seemed out of scope for this test.